app/src/main/java/jp/fluct/fluctsdk/exmaple/android/gmamediation/InterstitialActivity.kt

```diff
@@ -44,7 +43,6 @@ class InterstitialActivity : AppCompatActivity() {
 
             ad!!.loadAd(
                 AdRequest.Builder()
-                    .setGender(GENDER)
```

Comment:
NOTE:
This project was used GMA SDK `v16.x.x` previously, It was updated today: https://github.com/voyagegroup/GMA-Mediation-FluctSDK-Android/pull/8/files#diff-39e7d8c00954e920b98e7636f0ac30b2R49
`AdRequest.Builder::setGender` is deprecated in `v17.x.x`.